.gitignore

```
@@ -1,5 +1,37 @@
-/lib
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
 node_modules
-/coverage
-yarn-error.log
+.pnp
+.pnp.js
+.pnpm-store
+
+# testing
+coverage
+
+# next.js
+.next/
+out/
+
+# production
+build
 dist
+lib/lib
+
+# misc
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# local env files
+.env*.local
+
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
```

```suggestion
*.tsbuildinfo

```

Same for many other files :)